test.md

```diff
@@ -347,6 +347,15 @@ break the Markdown block up. This example does it with Markdown. You can't have
 break the Markdown block up.
 {: style="column-count: 3 "}
 
+## Running in-page Javascript
```

Comment:
I'm prepared to be convinced about that. I was not sure where people would go to look for this kind of info, and I think if it were me, I'd look in the `test.md` because I think of it as more of a "how-do-i".